app/src/components/dogs/DogDetail.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <section v-if="dog">
+  <h2> {{dog.name}} </h2>
+  <p>
+    {{dog.name}} is a type of {{dog.breed}}.
+  </p> 
+  <p> 
+    {{dog.name}} is {{dog.size}} that weighs {{dog.weight}} lbs.
+  </p>
+  <p>
+   It is {{dog.isadopted}} that {{dog.name}} was adopted into a great family! 
+  </p> 
+  <EditDog 
+    :dog="dog"
+    :onEdit="handleEdit"/>
+  <button @click="handleDelete"> REMOVE </button>
+  </section> 
+</template>
+
+<script>
+import api from '../../services/api'; 
+import EditDog from './EditDogs'; 
+
+
+export default {
+  data() {
+    return {
+      dog: null
+    };
+  }, 
+  components: {
+    EditDog
+  },
+  created() {
+    api.getDog(this.$route.params.id)
+      .then(dog => {
+        this.dog = dog; 
+      });
+  },
+  methods: {
+    handleDelete() {
+      api.deleteDog(this.dog.id)
+        .then(() => {
+          this.$router.push('/dog'); 
```

Comment:
```suggestion
          this.$router.push('/dogs'); 
```
I think redirect was a typo (intended to go back to dog list but was actually going to non-existent path).